app/Http/Transformers/ComponentsTransformer.php

```diff
@@ -45,7 +45,7 @@ public function transformComponent(Component $component)
                 'id' => (int) $component->company->id,
                 'name' => e($component->company->name),
             ] : null,
-            'notes' => ($component->notes) ? e($component->notes) : null,
```

Comment:
The component itself still has `notes` and I don't see a migration that changes that. And I think it's probably better to keep the component's "notes" as `notes` but the individual pivot note entries as `note` - that's a little bit clearer.

Reply:
Yeah, that was what I was going for. `notes` for the items themselves, `note` for pivots (including `action_logs`)